### ISSUE TITLE
Deploy Models using Google's Cloud Machine Learning Engine

### ISSUE THREAD

**Issue by slavakurilyak** [produvia/kryptos#65]
# User Story

As a machine learning developer, I want to deploy all predictive or classifications models on [Google's Cloud Machine Learning Engine](https://cloud.google.com/ml-engine/), so that I can deploy on several frameworks, such as:

1. scikit-learn for the breadth and simplicity of classical machine learning
2. XGBoost for the ease and accuracy of extreme gradient boosting
3. Keras for easy and fast prototyping of deep learning
4. TensorFlow for the cutting edge power of deep learning
